src/pybind/mgr/call_home_agent/module.py

```diff
@@ -0,0 +1,322 @@
+"""
+IBM Ceph Call Home Agent
+"""
+
+from typing import List, Any, Tuple, Dict, Optional, Set, Callable
+from datetime import datetime
+import json
+import requests
+import asyncio
+import os
+
+from mgr_module import Option, CLIReadCommand, MgrModule, HandleCommandResult, NotifyType
+
+from .config import get_settings
+from .options import CHES_ENDPOINT, INTERVAL_INVENTORY_REPORT_MINUTES, INTERVAL_PERFORMANCE_REPORT_MINUTES
+
+# Module with the functions to be used in telemetry and in CCHA for retrieving information from the ceph cluster
+from .dataCollectors import inventory
+
+reports_header = {
+    "agent": "RedHat_Marina_firmware_agent",
```

Comment:
I'm fairly sure this is a typo by Perry

```suggestion
    "agent": "RedHat_Marine_firmware_agent",
```

Reply:
RedHat?